cmd/kubeadm/app/apis/kubeadm/v1alpha3/doc.go

```diff
@@ -14,7 +14,83 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Package v1alpha2 is the package that contains the libraries that drive the kubeadm binary.
+// Package v1alpha3 is the API (config file) for driving the kubeadm binary.
+// Some of these options are also available as a command line flags, but
+// the preferred way to configure kubeadm is to pass a YAML file in with the
+// --config option.
+//
+// A fully populated example of the schema:
+//	apiVersion: kubeadm.k8s.io/v1alpha3
+//	kind: InitConfiguration
+//	etcd:
+//		# one of local or external
```

Comment:
i think double spaces need to be used here instead of tabs if someone wants to copy-paste this.